contracts/extensions/MultisigPermissions.sol

```diff
@@ -314,10 +315,25 @@ contract MultisigPermissions is ColonyExtensionMeta, ColonyDataTypes, GetActionS
     return domainSkillThreshold[_domainSkillId];
   }
 
+  function getMotionVoteThreshold(uint256 _motionId, uint8 _role) public view returns (uint256) {
+    Motion storage motion = motions[_motionId];
+    require(bytes32(1 << _role) & motion.requiredPermissions > 0, "multisig-role-not-required");
```

Comment:
Surprised to see this as a `require` vs simply a return -- don't we iterate through all roles elsewhere in the contract? Those don't return unnecessary errors?

Reply:
We do, and should skip over them entirely if they're not needed for the motion; this `require` shouldn't be being hit if this function is called from other contracts, but the function is public. You're proposing returning 0 in this case?

Reply:
That was my thought, but if the unnecessary permissions are being skipped elsewhere then I'm happy with the require as it is.